tests/robot-tests/run_tests.py

```diff
@@ -186,6 +190,14 @@
     robotArgs += ["--removekeywords", "name:operatingsystem.environment variable should be set"]
     robotArgs += ['--removekeywords',
                   'name:common.user goes to url']  # To hide basic auth credentials
+
+if args.custom_env: 
+    load_dotenv(args.custom_env)
+    assert os.getenv('PUBLIC_URL') is not None
+    assert os.getenv('ADMIN_URL') is not None
+    assert os.getenv('ADMIN_EMAIL') is not None
+    assert os.getenv('ADMIN_PASSWORD') is not None
```

Comment:
I'd move these asserts outside of the if/else so that you don't need to have them twice.
```
if args.custom_env:
  load_dotenv...
else args.blah
  load_dotenv..
assert ...
```